app/components/courses/entry_requirements_component.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-entry">Entry requirements</h2>
+  <h3 class="govuk-heading-m">Qualifications needed</h3>
+  <div data-qa="course__required_qualifications">
+    <p class="govuk-body">
+      <%= degree_grade_content(course) %>.
+    </p>
+
+    <% if course.degree_subject_requirements.present? %>
+      <p class="govuk-body">
+        <%= course.degree_subject_requirements %>
+      </p>
+    <% end %>
+
+    <p class="govuk-body">
+      <%= required_gcse_content(course) %>.
+    </p>
+
+    <p class="govuk-body">
+      <%= pending_gcse_content(course) %>.
+    </p>
+
+    <p class="govuk-body">
+      <%= gcse_equivalency_content(course) %>.
+    </p>
+
+    <% if course.additional_gcse_equivalencies %>
+      <p class="govuk-body">
+        <%= course.additional_gcse_equivalencies %>.
+      </p>
+    <% end %>
```

Comment:
When I was testing this locally, I noticed that one course showed a full stop on its own, because the `additional_gcse_equivalencies` field was blank rather than `null`. Think we should either check for presence, or coerce blanks to nulls at the Publish end or in the API?

Also, _hopefully_ providers will include their own full stops... 🤞  – if they don't, or if it's inconsistent, we could consider adding one if not already present (or making the guidance in Publish clearer).

```suggestion
    <% if course.additional_gcse_equivalencies.present? %>
      <p class="govuk-body">
        <%= course.additional_gcse_equivalencies %>
      </p>
    <% end %>
```